### DISCUSSION TITLE
gpg4usb key가 생성 안됩니다.

### DISCUSSION THREAD

**Discussion by sangdoli2771** [community/community#39241] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

다운하고 아이디 생성까지 되고 new key create 까지 뜨는데 나의 key 뜨지 않습니다.

**Comment by jfullstackdev** [comment]
I just tried it out, after successful creation, your page should look like this 

![image](https://user-images.githubusercontent.com/47092464/202181454-9571a75a-e3b1-4a1c-8944-f062f3f1a308.png)

**Reply by sangdoli2771** [reply]
다음과 같지 않습니다.
